ethics/21-distraction-fr.md

```diff
@@ -0,0 +1,373 @@
+Et force est de constater que les algorithmes modernes sont capables de prouesses spectaculaires,
+souvent très largement surhumaines,
+notamment grâce à la formalisation algorithmique des preuves mathématiques et de leur vérification.
+
+Et très clairement, à cette conférence, Big Tech était surreprésenté,
+qu'il s'agisse de Facebook, Google ou autres OpenAI,
+par opposition aux académiques qui n'ont généralement pas accès
+à suffisamment de supercalculateurs pour effectuer les calculs monstrueux
+nécessaires pour concevoir des algorithmes surhumains en mathématiques.
+https://www.ihp.fr/en/events/conference-flaim-formal-languages-ai-and-mathematics
+
+Qui plus est, à côté des dizaines de millions de dollars nécessaires
+pour payer la facture électrique des supercalculateurs qui font des mathématiques,
+les salaires cumulés des chercheurs présents se chiffraient eux aussi certainement en millions,
+voire en dizaines de millions.
+Mais donc, pourquoi ? Pourquoi Big Tech investit autant dans des domaines qui, clairement,
+ne sont pas les plus utiles pour leurs produits, et donc a priori pour leurs revenus ?
+
+Pourquoi Facebook, pourtant dans le dur financièrement,
+investit autant dans le machine learning pour les mathématiques ?
+Pourquoi Google a-t-il racheté DeepMind,
+et les félicite de faire des algorithmes
+pour bien jouer aux échecs, au go et aux jeux vidéos,
+ou pour résoudre le problème du repliement de protéine ?
+
+Et pourquoi OpenAI investit autant pour produire des IA "artistiques",
+capables de générer des images spectaculaires
+ou de converser de manière bluffante ?
+Qu'est-ce qui justifie, d'un point de vue économique, le salaire de ces chercheurs ?
+
+Telles sont les questions que je n'ai cessé de poser aux participants à la conférence.
+Et, en gros, personne ne s'était même posé la question.
+Tous étaient trop distraits par la spectacularité des algorithmes développés.
+Et bien, il me semble que c'était justement ça, l'objectif économique de cette conférence.
+Il s'agissait de distraire les chercheurs des enjeux économiques et géopolitiques
+des algorithmes de machine learning,
+et de donner l'impression que l'IA, c'est cool,
+voire que freiner son progrès spectaculaire serait immoral.
+
+(alors que nombres de leurs collègues, notamment Frances Haugen et Sophie Zhang,
+appellent désespérément à "ne pas fermer les yeux"
+sur les atrocités auxquels les algorithmes de Meta ont largement contribué,
+comme le génocide de Rohingyas et du Tigray, le chaos informationnel du COVID,
+voire les émeutes du Capitol et la tentative de coup d'état par l'extrême droite en Allemagne...)
+
+C'est ainsi que des étudiants en machine learning,
+qui ont beaucoup de remords à travailler pour Philipp Morris ou Total,
+sont heureux de candidater et de décrocher des postes à Google ou Facebook,
+voire que des membres du mouvement Altruisme Efficace ne jure que par DeepMind ou OpenAI,
```

Comment:
Les postes proposés par 80,000 hours dans ces 2 entreprises sont en alignement des IA, en modération ou en cybersécurité, je ne crois pas qu'il y en ait sur l'amélioration des performances. 

Mais comme toi, j'ai l'impression que DeepMind a fait plus de mal que de bien en précipitant l'arrivée de ces technologies. Je ne saurais pas trop dire pour OpenAI qui semble avoir quand même une grosse proportion d'employés sur la sécurité des IA (≈ 20% ?) (https://www.lesswrong.com/posts/mC3oeq62DWeqxiNBx/estimating-the-current-and-future-number-of-ai-safety). Et oui, c'est cool et donc ça donne plus envie d'y croire.

Reply:
C'est bien qu'il y ait des gens comme toi pour garder la tête froide et penser au gros des conséquences pour la société là où ce serait plus simple de suivre la hype.